redis/redis.go

```diff
@@ -187,3 +187,27 @@ type SlowLog struct {
 	// ClientName is the name set via the CLIENT SETNAME command (4.0 only).
 	ClientName string
 }
+
+// Latency represents a redis LATENCY LATEST
+type Latency struct {
+	// Latency event name
+	Event string
+
+	// Time is the unix timestamp at which the event was processed.
```

Comment:
This isn't a unix timestamp, same below.
```suggestion
	// Time of the latest latency spike for the event.
```